modules/govuk_jenkins/templates/jobs/copy_data_to_integration.yaml.erb

```diff
@@ -60,6 +70,11 @@
             HOSTNAME=deploy.integration.publishing.service.gov.uk
             API_KEY=<%= @ci_alphagov_api_key %>
             AUTH_TOKEN=<%= @auth_token %>
+      - slack:
+          team-domain: <%= @slack_team_domain %>
+          auth-token: <%= @environment_variables['SECOND_LINE_SLACK_AUTH_TOKEN']%>
```

Comment:
Ah OK, we're setting it as a global env var so that it can be accessed by all the different jobs using it without having to repeat it everywhere - I see! That makes sense. If this works then I'm fine with this.